### ISSUE TITLE
lpar.modify_resource fails if multiple new values are passed

### ISSUE THREAD

**Issue by jfhutchi** [pseries/rbvppc#4]
```
`def modify_resources(options, reboot = true)
    options.each do |key,val|
        if @@valid_attributes.include?(key)
            #Check for min/max/desired in the key to determine if
            #some bound needs to be checked first
            verify_and_handle_attr_bounds(options,key,val)

            set_attr_profile(val,key)

            #Handle setting of any instance variables that should change
            #due to this
            map_key_to_attr(key, val)
        end
    end 
    reactivate if reboot
end`
```

The current code loops through and changes one value at a time..This causes failures if the any of the old values don't fit into the ratio with the new values. We may need to update this to set all profile attributes at once instead of one at a time.

**Comment by jfhutchi** [comment]
hscroot@hmc:~> export LANG=en_US.UTF-8;chsyscfg -m test_frame -r prof -i "name=testlpar_profile, lpar_name=testlpar, min_proc_units=0.2"
An error occurred while changing the partition profile named testlpar_profile.
HSCL0622 The shared processor settings in the partition profile are not valid.

**Comment by jfhutchi** [comment]
#5 Linking pull request to issue

**Comment by woodc-ibm** [comment]
Merged #5, closing this issue
